collections/_projects/service-mesh-distributed-performance-management.md

```diff
@@ -6,53 +6,40 @@ In collaboration with Google, Layer5 is advancing the state of the industry thro
 author:
   twitter: layer5
 image:
-  path: /assets/images/buttons/smps.png
+  path: /assets/images/buttons/distributed-performance_white.svg
   width: 1200
-thumbnail: /assets/images/buttons/smps_white.svg
+thumbnail: /assets/images/buttons/distributed-performance_white.svg
 lang: en
 permalink:
-visibility: hide
 link: /service-mesh-distributed-performance-management
 redirect_from: distributed-performance
 
 ---
 <div class="row" id="smi" >
-  <div class="col m6">
+  <div class="col m8">
     <h3>
       Purpose and Overview
     </h3>
-    <p>The scope of this initiative includes all service mesh projects participating in the Service Mesh Interface specification. It’s important to acknowledge that conformance consists of both capabilities and compliance status. 
+    <p>Distributed load testing offers insight into system behaviors that arguably more accurately represent real world behaviors of services under load as that load comes from any number of sources. This project is being developed by Kush Trivedi as a part of Google Summer of Code intern with CNCF and Layer5.
```

Comment:
I see the same page being added above in the PR under _pages folder, why?

Reply:
@kushthedude good eye. This is part of the current site design that is something of a work around for the way that Jekyll and GitHub Pages handle trailing slashes on the end of URLs. I'm hopeful we can be right of the duplication at some point.